framework/src/main/java/org/checkerframework/framework/stub/StubParser.java

```diff
@@ -1204,6 +1185,18 @@ private void annotate(AnnotatedTypeMirror type, List<AnnotationExpr> annotations
         }
         for (AnnotationExpr annotation : annotations) {
             AnnotationMirror annoMirror = getAnnotation(annotation, allStubAnnotations);
+            if (annoMirror == null) {
+                // If the annotation was not imported, then #getAllStubAnnotations does
```

Comment:
This should be moved into `getAnnotation`.  Or is there some reason it can't be?